revscoring/languages/tests/test_persian.py

```diff
@@ -1,6 +1,6 @@
 from nose.tools import eq_
 
-from ..farsi import is_misspelled
+from ..persian import is_misspelled
 
```

Comment:
I don't know.  @Ladsgroup, what should the test case say when demonstrating a misspelling?